var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -92,11 +92,10 @@ class Openmpi(AutotoolsPackage):
 
     # Additional support options
     variant('java', default=False, description='Build Java support')
-    variant('sqlite3', default=False, description='Build sqlite3 support')
-    variant('vt', default=True,
-            description='Build support for contributed package vt')
+    variant('sqlite3', default=False, description='Build SQLite3 support')
+    variant('vt', default=True, description='Build VampirTrace support')
     variant('thread_multiple', default=False,
-            description='Enable MPI_THREAD_MULTIPLE support')
+            description='Enable multithreading support')
```

Comment:
Super minor comment: multithreading support in MPI comes in the form of:

- `MPI_FUNNELED`: only the `main thread` can make MPI calls
- `MPI_SERIALIZED`: multiple threads can make MPI calls, but one at a time
- `MPI_THREAD_MULTIPLE`: multiple threads can make MPI calls simultaneously

If I remember well this option activates the last one. The other two level of support are there by default.